_data/navigation.js

```diff
@@ -140,6 +140,13 @@ module.exports = {
           },
         ],
       },
+      {
+        section: 'ADDITIONAL INFORMATION',
+        contents: [
+          { url: '/docs/node-operators-faq/', title: 'FAQ' },
```

Comment:
Let's leave out the FAQ for now. There are some questions like "what is a phase" that should be answered in the Glossary/Terminology page though.